qa/standalone/mon/osd-pool-df.sh

```diff
@@ -0,0 +1,75 @@
+#!/usr/bin/env bash
+#
+# Copyright (C) 2017 Tencent <contact@tencent.com>
+#
+# Author: Chang Liu <liuchang0812@gmail.com>
+#
+# This program is free software; you can redistribute it and/or modify
+# it under the terms of the GNU Library Public License as published by
+# the Free Software Foundation; either version 2, or (at your option)
+# any later version.
+#
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU Library Public License for more details.
+#
+source $CEPH_ROOT/qa/standalone/ceph-helpers.sh
+
+function run() {
+    local dir=$1
+    shift
+
+    export CEPH_MON="127.0.0.1:7105" # git grep '\<7105\>' : there must be only one
```

Comment:
please don't reuse port number. and i'd recommend updating `src/test/mon/PGMap.cc` to exercise this fix.

Reply:
ooops....I forget to update port number. thank you

The logic is rather complicated, and existing unittest(replicated pool only) also misses `raw_used_rate` . so I add a standalone test. this standalone test is clear and straightforward.

adding some unittestes for EC pool to `src/test/mon/PGMap.cc` is necessary. I will do it.